index.html

```diff
@@ -43,6 +43,10 @@
     <p id="typewriter">https://
         <span id="hermes">foo</span>.js.org
     </p>
+    <p id="stats">
+        <span class="first">ACTIVE DOMAINS</span><span class="second" id="count">2755</span><br>
+        <span class="first">SOME EXAMPLES</span><span class="second" id="example"><a href="https://cluster-eventdispatcher.js.org">cluster-eventdispatcher.js.org</a></a></span>
```

Comment:
```suggestion
        <span class="first">ACTIVE DOMAINS</span><span class="second" id="count">~3000</span><br>
        <span class="first">SOME EXAMPLES</span><span class="second" id="example"></span>
```
We can make the count more generic here, so that is ages better if shown when the JS fails to execute.

No need to bring personal promotion into this and link to your subdomain by default, especially not for a domain that isn't even in cnames_active.js yet.